index.html

```diff
@@ -4,7 +4,71 @@
     <meta charset="UTF-8" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link href="style.css" rel="stylesheet" />
     <title>Personal Website</title>
   </head>
-  <body></body>
+
+  <body>
+    <header>
+      <h1>Adriana Morales Tremolada</h1>
+      <h2>Human</h2>
+      <nav>
+        <ul>
+          <li>About me</li>
+          <li>Interests</li>
+          <li>What the future holds</li>
+          <li>Contact</li>
+        </ul>
+      </nav>
+    </header>
+
+    <main>
+      <section>
+        <h2>About me</h2>
+      </section>
+      <section>
+        <h2>Interests</h2>
+      </section>
+      <section>
+        <h2>What the future holds</h2>
+      </section>
+      <section>
+        <h2>Contact</h2>
+      </section>
+    </main>
+
+    <footer>
+      <div>
+        <ul>
+          <li>About me</li>
+          <li>Interests</li>
+          <li>What the future holds</li>
+          <li>Contact</li>
+        </ul>
+      </div>
+      <div>
+        <h4>Let's connect:</h4>
+        <a href="" target="blank"
+          ><img
+            src="https://www.pngrepo.com/download/303212/xing-icon-logo.png"
+            alt="Xing Logo"
+        /></a>
+        <a href="" target="blank"
+          ><img
+            src="https://www.seekpng.com/png/detail/8-84419_linkedin-logo-png-icon-linkedin-logo-png.png"
+            alt="Linkedin Logo"
+        /></a>
+        <a href="https://github.com/adrianammt" target="blank"
+          ><img
+            src="https://image.flaticon.com/icons/png/512/25/25231.png"
+            alt="Github Logo"
+        /></a>
+        <a href="https://codepen.io/adrianamt" target="blank"
+          ><img
+            src="https://image.flaticon.com/icons/png/512/2111/2111501.png"
+            alt="Codepen Logo"
+        /></a>
```

Comment:
To put the links in a list would be perfect.

Reply:
Yes, will do.